pkg/backend/display/diff_test.go

```diff
@@ -78,6 +78,7 @@ func testDiffEvents(t *testing.T, path string, accept bool) {
 		ShowReplacementSteps: true,
 		ShowSameResources:    true,
 		ShowReads:            true,
+		ShowFullOutput:       true,
```

Comment:
This is a fine standard, but we should make sure there are tests for where this is false.

Reply:
@iwahbe @AaronFriel I've verified the following:

- diff code gets called when a new stack is initialized and destroyed
- diff code gets called when `pulumi up` is run and there is a new long output, a deleted long output, or a modified long output

I'm not sure why the tests are now failing; `ceb79f2` passed, but that printed the output value as is if there were fewer than three lines, even if one of the lines was > 150 chars. I also specified `true` for `ShowFullOutput` (line 81), meaning `truncatePropertyString` shouldn't be called, so the existing tests shouldn't be impacted